sessions/js-variables-and-numbers/social-media-post/css/styles.css

```diff
@@ -0,0 +1,30 @@
+*,
+*::before,
+*::after {
+	box-sizing: border-box;
+}
+
+body {
+	margin: 0;
+	font-family: -apple-system, BlinkMacSystemFont, 'Segoe UI', Roboto, Helvetica, Arial, sans-serif,
+		'Apple Color Emoji', 'Segoe UI Emoji', 'Segoe UI Symbol';
+	display: flex;
+	justify-content: center;
+	align-items: center;
+	height: 100vh;
+	background-color: #333;
+}
+
+header {
```

Comment:
Regarding best practices: do we want to use type selectors, or do we want to use class selectors exclusively (when/if possible)?

Reply:
This exercise is only concerned about the JS and not the HTML or CSS. 

The styling is only used for a note saying the students should open the JS console. There is no further code in HTML or CSS. I guess it's okay for this use case.

There are a couple of such exercises using this note.